src/components/App.js

```diff
@@ -1,10 +1,29 @@
 import React from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import Header from './Header';
+import Home from './Home';
 import Calculator from './Calculator';
-
+import Quote from './Quote';
 // eslint-disable-next-line
 const App = () => {
 
-  return (<div><Calculator /></div>);
+  return (
+    <div>
+      <Header />
+      <Router>
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/calculator" element={<Calculator />} />
+          <Route path="/quote" element={<Quote />} />
+        </Routes>
+      </Router>
+
+    </div>
```

Comment:
- [ ] To make your app act like a single page app, Please include the `<Header>` tag inside `<Router>` tag like the following:
```javascript
     <Router>
        <Header />
        <Routes>
          <Route path="/" element={<Home />} />
          <Route path="/calculator" element={<Calculator />} />
          <Route path="/quote" element={<Quote />} />
        </Routes>
      </Router>
```